cmd/node/multi_node.go

```diff
@@ -169,6 +169,9 @@ func getTestDefaultConfig(numOfInstances int) *config.Config {
 	cfg.LAYERS.RequestTimeout = 10
 	cfg.GoldenATXID = "0x5678"
 
+	// increase from 2s to 10s: for some reason, the multi node test requires a long timeout
+	cfg.SyncRequestTimeout = 10000
```

Comment:
i am unable to see where this variable is used in the codebase. is this still relevant?

Reply:
good catch! I cleaned this up. It may have been removed recently.